pkg/controller/controller.go

```diff
@@ -460,6 +460,9 @@ func (c *Controller) runAction(ctx context.Context, as *crv1alpha1.ActionSet, aI
 						Message: err.Error(),
 					}
 					ras.Status.Actions[aIDX].Phases[i].State = crv1alpha1.StateFailed
+					if output != nil {
+						ras.Status.Actions[aIDX].Phases[i].Output = output
+					}
```

Comment:
So if there are n number of phases and in the actionset status and I want to check the actual error log will have I have to get `status.phases[len(phases)-1].output.recentLogs`?

Reply:
`recentLogs` will only be added for the failed phase. Since the actionset would not continue the moment a phase failed, there should only be one `recentLogs` in the actionset's status..